aspnetcore/security/authentication/community-auth.md

```diff
@@ -0,0 +1,14 @@
+# Community OSS Auth options
+
+When developing ASP.NET Core applications you can use [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity) to secure your web applications. You can also replace that with another auth provider. This page contains the known open source auth options for ASP.NET Core. This page will be periodically updated as new providers become available.
```

Comment:
> This page contains the known open source auth options for ASP.NET Core

This page contains community provided open source authentication options for ASP.NET Core